database/migrations/2022_05_18_122013_add_location_geolocatie.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLocationGeolocatie extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::table('locations', function (Blueprint $table) {
+            $table->point('geolocatie')->after('prijsinfo')->nullable()->default(null);
+            $table->string('geolocatie_error')->after('geolocatie')->nullable()->default(null);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
```

Comment:
```suggestion
```

Reply:
See above

Reply:
ok, but only for migrations than ;P I have a the biggest hate boner you have ever seen against redundant or non-informative comments